components/03-modules/active-filters/ActiveFilters.vue

```diff
@@ -0,0 +1,160 @@
+<template>
+  <div>
+    <ul class="active-filters list">
+      <li
+        v-for="item in items"
+        :key="item.value"
+        class="active-filters__item list__item"
+      >
+        <alpaca-link
+          :href="item.href"
+          custom-class="active-filters__remove"
+          :aria-label="item.ariaLabel"
+          :title="item.icon.iconTitle"
+        >
+          <alpaca-icon
+            v-if="item.icon"
+            custom-class="active-filters__remove-icon"
+            :icon="item.icon.iconId"
+          />
+        </alpaca-link>
+
+        <span class="active-filters__label">
+          {{ item.label }} :&nbsp;
+        </span>
+        <span class="active-filters__value">
+          {{ item.value }}
+        </span>
+      </li>
+    </ul>
+
+    <alpaca-link
```

Comment:
It's probably going to be a button, no link, so there should be an option to define it somehow + listen for events, to actually clear those filters.